torch/_decomp/decompositions.py

```diff
@@ -1891,19 +1890,25 @@ def _to_copy(
         return x.clone()
     dtype_converted = False
     common_device = device_hint(x)
+    inputIsFake = isinstance(x, torch._subclasses.FakeTensor)
```

Comment:
nit: conventionally locals in Python use snake case, rather than camel case. The naming here is inconsistent even with the variables above and it sticks out like a sore thumb. Stick to local conventions.